gui/src/hooks/useOS.ts

```diff
@@ -0,0 +1,14 @@
+import * as os from "@tauri-apps/api/os";
+import { useEffect, useState } from "react";
+
+export function useOS() {
+  const [type, setType] = useState<string | undefined>();
```

Comment:
do we know the specific types this gets?
would be useful to use a more concrete type instead of a general string

Reply:
We can just use tauri's types 👍